SU-SWS/decanter
# Decanter 161  classes v02

# READY FOR REVIEW

# Summary
- Implementing functional Flexbox class-based grid system

# Steps to Test
1. Visit /section-core.html#kssref-core-grid in the style guide
1. Confirm that the grid performs as expected across all breakpoints, and in IE11

# Associated Issues and/or People
- [GitHub ticket](https://github.com/SU-SWS/decanter/projects/11#card-11056610)
